integration/proxy/proxy_test.go

```diff
@@ -1116,7 +1116,6 @@ func TestALPNProxyHTTPProxyBasicAuthDial(t *testing.T) {
 	}
 	cfg.Listeners = helpers.SingleProxyPortSetupOn(rcAddr)(t, &cfg.Fds)
 	rc := helpers.NewInstance(t, cfg)
-	defer rc.StopAll()
```

Comment:
Why remove this?

Reply:
Realized it was already deferred below after `rc.Start()`, but I guess I should remove the one below not this one

Reply:
Restored this and removed the one below